test/not-found-handler-rate-limited.test.js

```diff
@@ -82,21 +82,21 @@ test('Set not found handler can be rate limited with specific options', async t
   t.equal(res.statusCode, 404)
   t.equal(res.headers['x-ratelimit-limit'], '4')
   t.equal(res.headers['x-ratelimit-remaining'], '1')
-  t.equal(res.headers['x-ratelimit-reset'], '1')
+  t.ok(['0', '1', '2'].includes(res.headers['x-ratelimit-reset']))
```

Comment:
Why are the values not set to a fixed value?

Reply:
That is the flaky test I mentioned above, it fails sometimes on master too if there is enough time for the ttl to go down and get floored to 0 instead of 1

The [first check](https://github.com/fastify/fastify-rate-limit/blob/b867b5463e5bd342fa70687cb82fc95e2c29e6a2/test/not-found-handler-rate-limited.test.js#L79) of that test takes this into account, but the rest of the checks don't, so it would fail from time to time

Reply:
hmm. but it should be then never 2?

Reply:
Rarely it *is* 2 for the second test as well, the idea there is that it should be within that interval